LibXml2 is not needed to compile the gstreamer backend for Phonon

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -105,13 +105,6 @@ set_package_properties(GLIB2 PROPERTIES
 
 find_package(GObject REQUIRED)
 
-find_package(LibXml2 REQUIRED)
-set_package_properties(LibXml2 PROPERTIES
-    TYPE REQUIRED
-    DESCRIPTION "LibXml2"
-    PURPOSE "LibXml2 is required to compile the gstreamer backend for Phonon"
-    URL "http://xmlsoft.org/downloads.html")
-
 add_subdirectory(gstreamer)
 
 if(IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/po")
